getting-started/configuration/coldbox.cfc/configuration-directives/environments.md

```diff
@@ -27,6 +27,14 @@ environments = {
 };
 ```
 
+The regex match will also create a global setting called "environment" which you can access and use like this:
+
+```javascript
+if (controller.getSetting('environment') == 'development'){
```

Comment:
can you just make this use the façade please?

`if( getSetting( "environment" ) == "development" )`